inference-engine/src/snippets/src/op/broadcastload.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/op/broadcastload.hpp"
+
+#include <ngraph/runtime/reference/broadcast.hpp>
+
+using namespace std;
+using namespace ngraph;
+
+NGRAPH_RTTI_DEFINITION(snippets::op::BroadcastLoad, "BroadcastLoad", 0);
+
+snippets::op::BroadcastLoad::BroadcastLoad(const Output<Node>& x, Shape shape)
+: BroadcastMove(x, shape), broadcast_info(x.get_shape().size(), 0) {
+    constructor_validate_and_infer_types();
+}
+
+bool snippets::op::BroadcastLoad::visit_attributes(AttributeVisitor& visitor) {
+    return true;
+}
+
+std::shared_ptr<Node> snippets::op::BroadcastLoad::clone_with_new_inputs(const OutputVector& new_args) const {
```

Comment:
Please add `INTERNAL_OP_SCOPE` macros to operations.

Reply:
In general at the current moment we add CC macros (`INTERNAL_OP_SCOPE` for your case) to next methods: `clone_with_new_inputs`, `validate_and_infer_types`, `visit_attributes` and `evaluate`.
Can we add this macro to methods which don't have it?